src/util/hass-util.html

```diff
@@ -218,21 +232,17 @@
     case 'occupancy':
       return activated ? 'mdi:home' : 'mdi:home-outline';
     case 'opening':
-      return activated ? 'mdi:door-closed' : 'mdi:door-open';
-    case 'sound':
-      return activated ? 'mdi:music-note-off' : 'mdi:music-note';
-    case 'vibration':
-      return activated ? 'mdi:crop-portrait' : 'mdi:vibrate';
-    case 'gas':
-    case 'power':
-    case 'problem':
-    case 'safety':
-    case 'smoke':
-      return activated ? 'mdi:verified' : 'mdi:alert';
+      return activated ? 'mdi:crop-square' : 'mdi:exit-to-app';
```

Comment:
**Preference:**
Can we use `mdi:square` and `mdi:square-outline` here?

Reply:
I've never been a fan of the `mdi:crop-square` and `mdi:exit-to-app` either...

What about `mdi:radiobox-blank` and `mdi:radiobox-marked`? Then it keeps similar appearance with the default icons (`mdi:radiobox-blank` and `mdi:checkbox-marked-circle`).

Reply:
Personally I'd prefer not to use circle icons if the device class has already been specified as an opening. Especially since a filled radiobox looks like an interactable element. IMO an empty/filled square works well for the case when the user specified this is an opening, but none of the more specific types fit.

Reply:
Yeah, that makes sense, I agree. I just pushed a commit to address all of your comments.